-1 
Tested at: 7e429559bf62c6bd1c8a46d3af792f62d20a6c38
 I found an error when building: 
 
 <pre>Entering library rule at RecoMuon/MuonIdentification
>> Building LCG reflex dict from header file src/RecoMuon/MuonIdentification/src/classes.h
>> Compiling  /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-26-2300/src/RecoMuon/MuonIdentification/src/CSCTimingExtractor.cc 
In file included from input_line_9:54:
In file included from /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-26-2300/src/PhysicsTools/SelectorUtils/interface/MakePyVIDClassBuilder.h:4:
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-26-2300/src/PhysicsTools/SelectorUtils/interface/VersionedSelector.h:204:61: error: use of undeclared identifier 'cname'
    CINT_GUARD(plugin = CutApplicatorFactory::get()->create(cname,*icut));
                                                            ^
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_8_0_X_2015-10-26-2300/src/PhysicsTools/SelectorUtils/interface/VersionedSelector.h:17:26: note: expanded from macro 'CINT_GUARD'
#define CINT_GUARD(CODE) CODE
                         ^
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12110/9217/summary.html

--- a/PhysicsTools/SelectorUtils/interface/VersionedSelector.h
+++ b/PhysicsTools/SelectorUtils/interface/VersionedSelector.h
@@ -201,12 +201,12 @@ initialize( const edm::ParameterSet& conf ) {
       icut->getParameter<bool>("needsAdditionalProducts");     
     const bool ignored = icut->getParameter<bool>("isIgnored");
     candf::CandidateCut* plugin = nullptr;
-    CINT_GUARD(plugin = CutApplicatorFactory::get()->create(cname,*icut));
+    CINT_GUARD(plugin = CutApplicatorFactory::get()->create(name,*icut));
     if( plugin != nullptr ) {
       cuts_.push_back(SHARED_PTR(candf::CandidateCut)(plugin));
     } else {
       throw cms::Exception("BadPluginName")
-	<< "The requested cut: " << cname << " is not available!";
+	<< "The requested cut: " << name << " is not available!";
     }
     needs_event_content_.push_back(needsContent);
     const std::string therealname = realname.str();

